server/datastore/psql/create_maintenance_request.go

```diff
@@ -0,0 +1,16 @@
+package psql
+
+import (
+    "github.com/a2fumn2022/gpmn-nsl/server/datastore"
+)
+
+func (db *DB) CreateMaintenanceRequest(request datastore.MaintenanceRequest) (datastore.MaintenanceRequest, error) {
```

Comment:
I think I like it better if we returned a `*datastore.MaintenanceRequest` instead of the non-pointer type. That way on line 12 we could do `return nil, err` instead of returning the same request, since I think it doesn't make sense to return a request if there's an error

Reply:
fix: return type is now a pointer and now returns `nil, err`